docs/search/views/index.md

```diff
@@ -1,309 +1,304 @@
+
+The following table shows a comparison between search views and a full-text index:
+
+Feature                             | Search       | Full-text Index
+:-----------------------------------|:-------------|:---------------
+Term search                         | Yes          | Yes
+Prefix search                       | Yes          | Yes
+Boolean expressions                 | Yes          | Restricted
+Range search                        | Yes          | No
+Phrase search                       | Yes          | No
+Relevance ranking                   | Yes          | No
+Configurable Analyzers              | Yes          | No
+C8QL composable language construct  | Yes          | No
+Indexed attributes per collection   | Unlimited    | 1
+Indexed collections                 | Unlimited    | 1
+
+Search views guarantee the best execution plan (merge join) when querying multiple attributes.
+
+## Concepts
+
+A search view represents all documents available in a specified set of source collections. Each search view is an abstraction of some transformation applied to documents in the collections. The type of transformation is specific to the search view implementation and can be as simple as an identity transformation. 
+
+A search view combines Boolean and generalized ranking retrieval and ranks each Boolean-approved document. For ranking text retrieval, we use the Vector Space Model (VSM) which uses documents and queries to represent vectors in a space formed by the _terms_ of the query. A term can include single words, keywords, and phrases. You can use [Analyzers](#analyzers) to boost value analysis with tokenization.
+
+The document vectors that are closer to a query vector are more relevant. The closeness is expressed as the cosine of the angle between two vectors ([cosine similarity](https://en.wikipedia.org/wiki/Cosine_similarity)). We evaluate the following expression to define how relevant document `d` is to query `q`:
+
+`cos a = (d * q) / (|d| * |q|)`
+
+- `d * q` is the dot product of the query vector `q` and document vector `d`
+- `|d|` is the norm of vector `d`
+- `|q|` is the norm of vector `q`
+
+The vector components must be computed up front. Since space is formed by terms, you can use _term weights_ as coordinates. The following probability and statistical weighting models are implemented in C8Search:
+
+- [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25)
+- [TF-IDF](https://en.wikipedia.org/wiki/Tf%E2%80%93idf)
+
+Both models rely on two components:
+
+- _Term frequency_ (TF): The number of times term `t` appears in document `d`.
+- _Inverse document frequency_ (IDF): How common or rare the term is across all documents.
+
+Searching and ranking capabilities are provided by the [IResearch library](https://github.com/iresearch-toolkit/iresearch)
+
+## Integration
```

Comment:
Would it make sense to take these and the following h2s and make them separate sections under Search Views?

Reply:
@diana-macrometa Sounds good, would definitely make the intro more approachable.